editor.planx.uk/src/ui/shared/DataTable/components/MultipleOptionSelectFilter.tsx

```diff
@@ -58,7 +58,6 @@ export function MultipleOptionSelectFilter<T>(props: Props<T>) {
           setChipData(value);
           return applyValue({ ...item, value });
         }}
-        value={item.value}
```

Comment:
Removed as this was throwing the "controlled" / "uncontrolled" input react error.